templates/chat-template/app/api/auth/new-user/route.ts

```diff
@@ -1,52 +1,43 @@
-import { hashPassword } from "@/lib/crypt.js"
+import { hashPassword } from "@/lib/crypt.js";
+import { TriplitClient } from '@triplit/client';
+
+const client = new TriplitClient({
```

Comment:
I think we should use HttpClient directly here. Creating a `TriplitClient` has some side effects such as setting up a WebSockets connection with the provided parameters, and has a cache. Neither of those are needed here, and we're not using the client elsewhere in the app, so better to use HttpClient which is just a thin wrapper around `fetch`.